src/frontend/src/handler/create_table.rs

```diff
@@ -1312,24 +1312,22 @@ pub async fn handle_create_table(
                 .await?;
         }
         Engine::Iceberg => {
-            // export AWS_REGION=your_region
-            // export AWS_ACCESS_KEY_ID=your_access_key
-            // export AWS_SECRET_ACCESS_KEY=your_secret_key
-            // export RW_S3_ENDPOINT=your_endpoint
-            // export META_STORE_URI=your_meta_store_uri
-            // export META_STORE_USER=your_meta_store_user
-            // export META_STORE_PASSWORD=your_meta_store_password
-
-            let s3_endpoint = if let Ok(endpoint) = std::env::var("RW_S3_ENDPOINT") {
+            let s3_region = if let Ok(region) = std::env::var("AWS_REGION") {
+                region
+            } else {
+                "us-east-1".to_string()
+            };
```

Comment:
Should this be a required field without default value?

Reply:
You are right. Currently, the default value is used in local development. We can make all of them required.